### ISSUE TITLE
Peer review from Ellen

### ISSUE THREAD

**Issue by ellenhsieh** [css-research/fp-siyiii#3]
Thank you for presenting this interesting topic on faster image rating. The outlay of the poster is neat. The arrangement of the sections is nice. Most of the contents use the bullet point to make the poster looks organized, which can help audience to get the idea in each section quickly.

As for the design of the poster, the red border of each section seems not so necessary. It would look nicer to omit that ref border and maybe change the color of the brown color parts into some lighter color (or maybe Uchicago red). It is really nice to use bold font to emphasize the critical idea. The colors of plots are really nice and easy to tell the differences between the rating time. Using the consistent colors (red and blue) makes the posters looks nice. The words are not too crowded and clear to read. However, the font size seems differ from different part, maybe can try to use same font size for regular texts and maybe bigger one or bold to emphasize the point you think is important.

As for the contents for background section, some bullet points seems more like describing a general idea, for instance, machine learning requires a massive dataset. It would be nice to describe more about the background knowledge for your study. Some general ideas might not be that necessary, or maybe they can be put after the important background knowledge. 

In conclusion, this is a nice poster for presenting this interesting topic. Thank you fro sharing such nice knowledge and idea for us!

**Comment by siyiii** [comment]
Thanks for your suggestions, Ellen!
